android/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
 rootProject.allprojects {
     repositories {
         google()
-        maven {
-            url 'https://sdk.squareup.com/public/android'
```

Comment:
Why are you removing our repo? This is where we host the IAP SDK artifacts

Reply:
Does it compile without reference to our repo?

Reply:
Sorry it was mistake 
I changed it